docs/api/README.md

```diff
@@ -0,0 +1,253 @@
+# Flipper::Api
+
+API for the [Flipper](https://github.com/jnunemaker/flipper) gem.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+    gem 'flipper-api'
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install flipper-api
+
+## Usage
+
+`Flipper::Api` is a mountable application that can be included in your Rails/Ruby apps. In a Rails application, you can mount `Flipper::Api` to a route of your choice:
+
+```ruby
+# config/routes.rb
+YourRailsApp::Application.routes.draw do
+  mount Flipper::Api.app(flipper) => '/flipper-api'
+end
+```
+
+For more advanced mounting techniques and for suggestions on how to mount in a non-Rails application, it is recommend that you review the [`Flipper::UI` usage documentation](https://github.com/jnunemaker/flipper/blob/master/docs/ui/README.md#usage) as the same approaches apply to `Flipper::Api`.
```

Comment:
I like linking to UI. Someday maybe we can just have a mounting.md file that docs both of them, but this works for me!